src/components/Figure.js

```diff
@@ -2,20 +2,31 @@ import React from 'react';
 import styled from '@emotion/styled';
 import CaptionLegend from './CaptionLegend';
 import Image from './Image';
+import { getImageAlignmentStyle } from '../utils/get-image-alignment-style';
 import { getNestedValue } from '../utils/get-nested-value';
 import { size } from './dev-hub/theme';
 
 const Figure = styled('figure')`
     margin: 0 0 ${size.articleContent};
+    width: ${({ figWidth }) => figWidth};
+    ${({ alignStyle }) => alignStyle};
 `;
-export default ({ nodeData, ...rest }) => (
-    <Figure
-        className="figure"
-        style={{
-            width: getNestedValue(['options', 'figwidth'], nodeData) || 'auto',
-        }}
-    >
-        <Image nodeData={nodeData} captioned={!!nodeData.children.length} />
-        <CaptionLegend {...rest} nodeData={nodeData} />
-    </Figure>
-);
+
+const removeAlignOption = nodeData => delete nodeData['options']['align'];
+
+export default ({ nodeData, ...rest }) => {
+    const customAlign = getNestedValue(['options', 'align'], nodeData);
+    if (customAlign) {
+        // Apply the alignment to the figure, no need to also apply to image
+        removeAlignOption(nodeData);
```

Comment:
We don't need to pass this to image as well now. If we do it is fine but would rather just deal with alignment here.

Reply:
Actually we still do, going to undo this change.